src/components/App.tsx

```diff
@@ -70,7 +72,27 @@ export default function App({
       hideConsentManager: () => handleSetViewState('close'),
       toggleConsentManager: () =>
         handleSetViewState(viewStateIsClosed(viewState) ? 'open' : 'close'),
-      autoShowConsentManager: () => !confirmed && handleSetViewState('open'),
+      autoShowConsentManager: () => {
+        const privacySignals = airgap.getPrivacySignals();
+        const applicablePrivacySignals =
+          privacySignals.has('DNT') ||
+          (privacySignals.has('GPC') && !airgap.getRegimes().has('GDPR'));
+        const shouldShowNotice = !confirmed || applicablePrivacySignals;
+        if (!shouldShowNotice) {
```

Comment:
I'm not sure about this actually - if we're going to add this I would also write docs to make sure we explain to the customer why it's not working as they expected

@bencmbrook what do you think?

Reply:
Those are good points. I definitely expect to need to update this approach as the legal field changes.

I'm going to refactor the auto-prompt suppression logic to be a bit more clear and easier to maintain. My next commit will check `DNT || GPC_BANNER_SUPPRESSION_REGIMES. includes(regime)` where that constant is just `['CPRA']` for now

Reply:
I've updated my approach to dynamically resolve the current consent selections, and only suppress auto-prompting with GPC when SaleOfInfo is the only available consent selection

Reply:
Once we have https://github.com/transcend-io/website-2.0/pull/41 merged, I will have a URL to point to for the docs regarding this behavior as well.

Reply:
I added a docs link, dependent on the above PR merging